pkg/swap/swap.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
Out of curiosity, why? I don't care all that much (other than having to sh-ify the script), but lighter and easier to install bash than, say, python or ruby or node or whatever.